src/tests/ftest/network/zero_config.py

```diff
@@ -87,7 +87,9 @@ def get_log_info(self, hosts, dev, env_state, log_file):
             bool: status of whether correct device was used.
 
         """
-        cmd = "head -50 {}".format(log_file)
+	# anticipate log switch
+        cmd = "if [ -f {0}.old ]; then head -50 {0}.old; else head -50 {0}; fi"
+              .format(log_file)
```

Comment:
(pylint-syntax-error) unexpected indent

Reply:
indenting looks wrong here, no?

Reply:
Oops, and I have forgotten the continuation char